### DISCUSSION TITLE
Allure Report Preview

### DISCUSSION THREAD

**Discussion by spags-lacework** [allure-framework/allure2#2413] [Questions & Support]
Does Allure have a way to get a preview of the report? 
For example
```
99/100 tests passed.
Tests Failing:
FooBarTest
```
Then I can link to the full details of a report.
?

**Comment by baev** [comment]
Console reporter is planned for Allure 3

**Reply by spags-lacework** [reply]
What's the ETA on Allure 3?

**Reply by baev** [reply]
We expect alpha builds to be available next month.

**Reply by noomorph** [reply]
@baev, are there many changes planned for the `-container.json` and `-result.json` format? 😅

**Reply by baev** [reply]
@noomorph a few. However, we are not planning to push results format changes any time soon.

**Comment by piyush466** [comment]
![image](https://github.com/allure-framework/allure2/assets/111576602/17fd8e14-7813-4495-bb6e-556866f7c02e)
Facing this issue 
Please help
